code/game/objects/items/ornaments.dm

```diff
@@ -0,0 +1,187 @@
+/obj/item/ornament
+	name = "ornament"
+	desc = "A glass ornament. What some would call plain, others call elegant."
+	icon = 'icons/obj/ball_ornaments.dmi'
+	icon_state = "white_ball_ornament"
+	w_class = 2
+
+/obj/item/ornament/attackby(obj/item/weapon/W, mob/user)
+	..()
+	if(istype(src, /obj/item/ornament/teardrop))
```

Comment:
You mean with numeric indices or assoc indices?

For numeric sadly not, but for assoc you can just do `list("x" = 10, "c" = 20)`